_scss/defold.scss

```diff
@@ -23,6 +23,19 @@
 ******************************************************************************/
 
 .dark-mode {
+	.dark\:darken-10 {
+		background-blend-mode: darken;
+		background-color: rgba(0, 0, 0, 0.10);
+	}
+
+	.learn-section {
+		padding: 0;
+
+		& > div {
+			padding: 8rem 0 7rem;
+		}
+	}
```

Comment:
What is the purpose of this?

Reply:
Heh, it changes nothing. At one point it was useful, not anymore. Deleted it.